File: crates/deslop-mcp/tests/fixtures/dart-mcp/order_totals.dart
double computeOrderTotal(List<OrderLine> lines, double levy, double rebate) {
  var running = 0.0;
  for (final line in lines) {
    final entryTotal = line.price * line.count;
    if (line.discounted) {
      running = running + entryTotal * 0.9;
    } else {
      running = running + entryTotal;
    }
  }
  final withLevy = running * (1 + levy);
  final afterRebate = withLevy - rebate;
  return afterRebate < 0 ? 0 : afterRebate;
}

File: crates/deslop-mcp/tests/fixtures/fsharp-mcp/order_totals.fs
module OrderTotals

let computeOrderTotal (lines: OrderLine list) (levy: float) (rebate: float) : float =
    let mutable running = 0.0
    for line in lines do
        let entryTotal = line.Price * line.Count
        if line.Discounted then
            running <- running + entryTotal * 0.9
        else
            running <- running + entryTotal
    let withLevy = running * (1.0 + levy)
    let afterRebate = withLevy - rebate
    if afterRebate < 0.0 then 0.0 else afterRebate
